src/actionModifiers/ImportExportWithDatasets/importWithDatasetsModifier.js

```diff
@@ -9,6 +9,7 @@ function importWithDatasetsModifier(openmct) {
     let datasets;
     let referencedDatasets;
     let component;
```

Comment:
I don't think this is used anymore?

Reply:
@jvigliotta , what isn't used anymore, exactly?

Reply:
I thought `component` but I was wrong, I'll resolve this one!